### PR TITLE
DOMA-1750 [Frontend] Temporary auth flow remake

### PR DESCRIPTION
**Pull request by MrFoxPro**

### CODE REVIEW

File: apps/condo/pages/auth/forgot.tsx [hunk 1]
```diff
@@ -39,10 +42,15 @@ const ResetPage: AuthPage = () => {
     const PhoneMsg = intl.formatMessage({ id: 'pages.auth.register.field.Phone' })
     const ExamplePhoneMsg = intl.formatMessage({ id: 'example.Phone' })
 
+    const [form] = Form.useForm()
+    const { executeRecaptcha } = useGoogleReCaptcha()
+    const { token, setToken, setPhone } = useContext(RegisterContext)
+
+    const [step, setStep] = useState<'inputPhone' | 'validatePhone'>('inputPhone')
```
Comment (AntonAL) [inline apps/condo/pages/auth/forgot.tsx:49]: Please, extract it to TypeScript type

**Review by AntonAL — COMMENTED** [review]
Roll it out with backend, it's hard to test

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/pages/auth/forgot.tsx [hunk 2]
```diff
@@ -39,10 +41,16 @@ const ResetPage: AuthPage = () => {
     const PhoneMsg = intl.formatMessage({ id: 'pages.auth.register.field.Phone' })
     const ExamplePhoneMsg = intl.formatMessage({ id: 'example.Phone' })
 
+    const [form] = Form.useForm()
+    const { executeRecaptcha } = useGoogleReCaptcha()
+    const { token, setToken, setPhone } = useContext(RegisterContext)
+
+    type StepType = 'inputPhone' | 'validatePhone'
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:48]: move type definition out of component pls

File: apps/condo/pages/auth/forgot.tsx [hunk 3]
```diff
@@ -99,69 +106,84 @@ const ResetPage: AuthPage = () => {
...
         })
     }
+    const Content = () => {
+        if (isLoading) {
+            return <Loader size="large" />
+        }
+        return (
+            <>
+                <Col span={24}>
+                    <Typography.Title style={{ textAlign: 'left' }}>{ResetTitle}</Typography.Title>
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:116]: use const styles

File: apps/condo/pages/auth/forgot.tsx [hunk 4]
```diff
@@ -99,69 +106,84 @@ const ResetPage: AuthPage = () => {
...
+                </Col>
+                <Col span={24}>
+                    <Typography.Paragraph style={{ textAlign: 'left' }}>{InstructionsMsg}</Typography.Paragraph>
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:119]: use const styles

File: apps/condo/pages/auth/forgot.tsx [hunk 5]
```diff
@@ -99,69 +106,84 @@ const ResetPage: AuthPage = () => {
...
+                </Col>
+                <Col span={24}> 
+                    <Form
+                        {...FORM_LAYOUT}
+                        form={form}
+                        name='forgot-password'
+                        validateTrigger={['onBlur', 'onSubmit']}
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:126]: use const array

File: apps/condo/pages/auth/forgot.tsx [hunk 6]
```diff
@@ -99,69 +106,84 @@ const ResetPage: AuthPage = () => {
...
+                                            return (
+                                                <Button
+                                                    onClick={() => {
+                                                        form.validateFields().then(() => {
+                                                            runAction()
+                                                        }).catch(_ => {
+                                                        // validation check failed - don't invoke runAction
+                                                        })
+                                                    }}
+                                                    type={isCountDownActive ? 'sberGrey' : 'sberPrimary'}
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/auth/forgot.tsx:156]: move function to const pls

File: apps/condo/pages/auth/forgot.tsx [hunk 7]
```diff
@@ -110,65 +106,74 @@ function ResetPageView () {
...
+                                        {({ countdown, runAction }) => {
+                                            const isCountDownActive = countdown > 0
+                                            return (
+                                                <Button
+                                                    onClick={() => {
+                                                        form.validateFields().then(() => {
+                                                            runAction()
+                                                        }).catch(_ => {
+                                                        // validation check failed - don't invoke runAction
+                                                        })
```
Comment (AntonAL) [inline apps/condo/pages/auth/forgot.tsx:154]: Maybe to write something to console here, it will be sent to Sentry for further debugging

**Review by akarjakin — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]
LGTM!

**Review by toplenboren — APPROVED** [review]
LGTM!

**Pull request merged**